src/components/Profile/VotedProposalsBox.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useState } from 'react'
+
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import { Container } from 'decentraland-ui/dist/components/Container/Container'
+
+import useVotedProposals from '../../hooks/useVotedProposals'
+import FullWidthButton from '../Common/FullWidthButton'
+import ProfileProposalItem from '../Proposal/ProfileProposalItem'
+
+import { ProfileBox } from './ProfileBox'
+import './VotedProposalsBox.css'
+
+interface Props {
+  address: string
+}
+
+const PROPOSALS_AMOUNT = 5
+
+function VotedProposalsBox({ address }: Props) {
+  const t = useFormatMessage()
+  const [skip, setSkip] = useState(0)
+  const { votes } = useVotedProposals(address, PROPOSALS_AMOUNT, skip)
```

Comment:
What if this hook returns a `viewMore` function and `votes` can handle this update so you can avoid doing this logic here? :)